src/AbstractOperations/computations.jl

```diff
@@ -37,17 +39,20 @@ Perform a `computation`. The result is stored in `computation.result`.
 function compute!(computation::Computation)
     arch = architecture(computation.result)
     result_data = data(computation.result)
-    @launch(device(arch), config=launch_config(computation.grid, :xyz),
-            _compute!(result_data, computation.grid, computation.operation))
+
+    workgroup, worksize= work_layout(computation.grid, :xyz)
+    kernel! = _compute!(device(arch), workgroup, worksize)
```

Comment:
So this is a tradeoff. If your input sizes stay constant for these kernels during the runtime of the program this is okay! Otherwise you are recompiling the kernel for each unique pair.

Reply:
For the vast majority of uses, input sizes are constant during runtime because they are statically determined by the grid (and nobody has tried to run two models with different grids at the same time --- yet). 

Is there are performance benefit to defining kernel size statically?